src/components/ScenarioParameters/components/ScenarioParametersTabs.js

```diff
@@ -40,86 +40,71 @@ const useStyles = makeStyles(theme => ({
 }));
 
 const ScenarioParametersTabs = ({
```

Comment:
Was thinking about, this component, with little work it could be "generalized" :thinking: right now it's not useful, maybe when we'll work on wizard component...  We'll see ;)

Reply:
Yes, this component defines the layout of the scenario parameters tabs, we can imagine adding support for more layouts in the future.